### ISSUE TITLE
Get Tags using API while subscribing?

### ISSUE THREAD

**Issue by surjithctly** [spatie/mailcoach-support#274]
How to get tags while subscribing via API?

I have tried the following but not working.

```json
{
    "email": "john3@doe.com",
    "tags" : "beta"
}
{
    "email": "john3@doe.com",
    "tags" : ["beta", "alpha"]
}

{
    "email": "john3@doe.com",
    "tags" : "beta;alpha"
}
```

**Comment by surjithctly** [comment]
@freekmurze Can anyone look at this issue?

**Comment by freekmurze** [comment]
I don't understand the question. Do you want to set the tags when subscribing?

**Comment by electronick86** [comment]
I think that we cannot create a subscriber with tags and extra-attributes using the API. https://mailcoach.app/docs/v3/package/api/subscribers#subscribing-to-an-email-list 

Correct?

**Comment by freekmurze** [comment]
Right! I'd accept a PR with tests and updated docs that adds this to the package. If none such PR comes in, we'll take a look at it in the next few weeks.

**Comment by surjithctly** [comment]
Yes. @electronick86 was right.

**Comment by riasvdv** [comment]
This is now available in 3.3.0, `tags` and `extra_attributes` expect an array
